Integrations/Elasticsearch_v2/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
 ##[Unreleased]  
+Added support for timestamp in Elasticsearch v2.
+   - if the Time Field for fetch is a timestamp simply check the relevant box in the integration setup
+    
+## [19.9.1] - 2019-09-18
```

Comment:
Why did you add a release version?

Reply:
wasn't I supposed to? - the notes below were already released.

Reply:
nope we didn't, this will require manual fixing for now, the actual fix for the issue will be in the next clear. (today)